### PR TITLE
New effect arguments

### PR DESCRIPTION
**Pull request by starkillerOG**
**Description:**
see PR in home-assistant below

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13500

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/light.hyperion.markdown [hunk 1]
```diff
@@ -33,3 +33,13 @@ Configuration variables:
 - **hdmi_priority** (*Optional*): The priority of the HDMI grabber of this Hyperion instance, note that this priority must be higher than all other priorities used for correct behavior. Defaults to `880`.
 - **default_color** (*Optional*): The color of the light. Defaults to `[255, 255, 255]`.
 - **effect_list** (*Optional*): The list of effects that can be used. Defaults to `['HDMI', 'Cinema brighten lights', 'Cinema dim lights', 'Knight rider', 'Blue mood blobs', 'Cold mood blobs', 'Full color mood blobs', 'Green mood blobs', 'Red mood blobs', 'Warm mood blobs', 'Police Lights Single', 'Police Lights Solid', 'Rainbow mood', 'Rainbow swirl fast', 'Rainbow swirl', 'Random', 'Running dots', 'System Shutdown', 'Snake', 'Sparks Color', 'Sparks', 'Strobe blue', 'Strobe Raspbmc', 'Strobe white', 'Color traces', 'UDP multicast listener', 'UDP listener', 'X-Mas']`.
+
+A few notes:
+
+- When an effect is called with the light.turn_on service, aditional arguments to the effect can be specified using the following syntax:
```
Comment (frenck) [inline source/_components/light.hyperion.markdown]: I don't wanna be picky about something...
But you sate `A few notes:`, is followed by a list containing one single note?

Comment (frenck) [reply source/_components/light.hyperion.markdown]: Also `aditional ` -> `additional`

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks, @starkillerOG. Could you please take a look at my comments? 👍

File: source/_components/light.hyperion.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_components/light.hyperion.markdown]: Thanks for your feedback!
I made some changes and added some aditional information.
Could you have a second look at it?

**Comment by starkillerOG** [comment]
@frenck I made changes based on your comments and added some additional links for more information. Could you have a second look?

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
@starkillerOG, Thanks for updating!
Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing this PR since the parent PR has been closed as well.

**Pull request closed without merging**